posts/fakes.md

```diff
@@ -0,0 +1,284 @@
+<meta name="id" content="7799320222051028141">
+<meta name="labels" content="testing,object oriented programming,java,microservices,domain-driven design">
+<meta name="title" content="The secret world of testing without mocking: domain-driven design, fakes, and other patterns to simplify testing in a microservices architecture">
+<meta name="description" content="How to simplify microservice testing through a few simple 
+applications of domain-driven design and in-memory test doubles (fakes).">
+
+Much has been said about mocks, the controversial, swiss-army knife of test doubles. Mocking has
```

Comment:
Still reviewing this, but do you think there is value in doing a before and after in code? That's how I embraced this concept in the first place. We were trying to implement a test utility for ProductService that could be reused across different tests. But it was using Mockito and we switched it out to use Fakes. And it was like the fog suddenly cleared.

Reply:
Yeah I agree, not enough examples. I might try and adapt from some real life ones like that.